react/OrderItems.tsx

```diff
@@ -42,6 +42,15 @@ const isSameItem = (
   const isSameId = input.id?.toString() === item.id
   const isSameSeller = input.seller === item.seller
 
+  /**
+   * If item is a gift, it should not be considered as the same item of the added one,
+   * otherwise buyer will not see two different products in the cart (the gift and the paid one)
+   *  */
+  // @ts-expect-error isGift has to be typed on vtex.checkout-graphql
```

Comment:
Just a heads up: after deploying `checkout-graphql`, run `vtex setup --typings` to update the GraphQL typings and remove this line.

Reply:
Sure! I will keep an eye out for when `checkout-graphql` is released to update it.

Reply:
@filafb done!